common/changes/@uifabric/experiments/v-vibr-ShimmerComponent_2018-03-06-18-42.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
@Vitalius1 are you working with someone on this? It would be great to have pair ownership to ensure we have someone who can more deeply provide feedback and shared fixing.

Reply:
Yes I am. Mainly I get feedback from @atneik .

Reply:
@atneik can you review please?

Reply:
@atneik can you review please?